templates/zerver/help/make-an-announcement.md

```diff
@@ -0,0 +1,38 @@
+# Making an Announcement
+
+An announcement on Zulip is a message which will reach
+ all users organization-wide.
+Announcements on Zulip are sent using the `#announce` stream.
+
+1. To send an announcement, click the **New stream message**
+ button located at the bottom of your screen.
+ A box similar to the one shown in the below image will appear.
+
+    ![New stream message](/static/images/help/new-stream.png)
+
+3. In the **Stream** field, input `announce` to select the `#announce` stream.
+
+4. Enter the topic for your announcement in the **Topic** field.
+
+    !!! tip ""
+        Your topic name can be no longer than 52 characters.
+
+5. Enter your announcement in the box labeled
+ **Compose your message here...** or messaging box.
```

Comment:
Since you're using it later in the text, instead of `or messaging box` I'd use `(compose box)` - see the [glossary](https://github.com/zulip/zulip/blob/master/templates/zerver/help/zulip-glossary.md) for naming conventions.